### DISCUSSION TITLE
Module Creator

### DISCUSSION THREAD

**Discussion by markagama** [oqtane/oqtane.framework#1717] [Q&A]
I'm following the guide on how to create a module https://blazorhelpwebsite.com/ViewBlogPost/45
I selected module creator and in the Pane option, I cannot see an option for Centre 100% Pane - should there be one? I can see Top 100% Pane and Bottom 100% Pane. I need to place a control that would use all of the available space. In this instance it appears logical to select Top 100% Pane.

**Comment by ADefWebserver** [comment]
Oqtane has changed the pane options since the tutorial was created. 
You will want to experiment with all the avaliable existing options to find one that works for you.
